libraries/Microsoft.Bot.Builder.AI.LUIS/Microsoft.Bot.Builder.AI.Luis.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageId>Microsoft.Bot.Builder.AI.Luis</PackageId>
     <Description>LUIS Middleware and Recognizers for the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using LUIS.</Summary>
+    <ContentTargetFolders>content</ContentTargetFolders>
```

Comment:
Did you mean to add this?

Reply:
yes, that should have been there and wasn't